index.bs

```diff
@@ -1187,7 +1183,7 @@ The {{FileReader}} object must be the event target for all events in this specif
 
 When this specification says to <dfn>fire a progress event</dfn> <i>called e</i>
 (for some {{ProgressEvent}} <code>e</code>
-at a given {{FileReader}} <code>reader</code> as the <a>context object</a>),
+at a given {{FileReader}} <code>reader</code> as <a>this</a>),
```

Comment:
Just because "as this" reads poorly, how about just dropping that part of the sentence? I don't think it adds anything.

Reply:
I think it's meant to say that when the spec says "fire a progress event" you have to use this as the target? But this could do with a rewrite.

I'm happy to drop "as this" though, but I'm not sure if that helps or hurts. (I'm not signing up for the bigger fix.)

Reply:
dropping "as this" sounds good to me, agree that this could do with a rewrite but that indeed seems like a separate issue.